app/views/wiki/new.html.erb

```diff
@@ -29,6 +29,14 @@ See COPYRIGHT and LICENSE files for more details.
 
 <% html_title t("create_new_page") %>
 
+<%=
+  render Primer::OpenProject::PageHeader.new do |header|
+    header.with_title { t("create_new_page") }
```

Comment:
That string tranlsates to "Wiki page", maybe we should change it to "Create new wiki page"?